data/movies_crud.rb

```diff
@@ -0,0 +1,20 @@
+require 'json'
+require_relative '../classes/movie'
+
+def load_movies
+  movies_json = File.read('./data/movies.json')
+  movies_data = []
+  if movies_json.empty?
+    movies_data
+  else
+    JSON.parse(movies_json).each do |json|
+      movies_data << Movie.from_json(json)
+    end
+  end
+  movies_data
+end
+
+def save_movies(movies)
+  movies_data = JSON.generate(movies)
+  File.write('./data/movies.json', movies_data)
+end
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/164894132-f062f7b7-0c5f-45c8-9232-d7e71c49288a.png)

- [ ] Kindly make sure your app does not crush when any of your `JSON` file is not present. Hint :point_down: 

```rb 
# Kindly not that this is only an example you can solve it the way you can 
def load_movies
  movies_data = []
  if File.exist?('./data/movies.json')
    movies_json = File.read('./data/movies.json')
    if movies_json.empty?
      movies_data
    else
      JSON.parse(movies_json).each do |json|
        movies_data << Movie.from_json(json)
      end
    end
    movies_data
  else 
    File.write('./data/movies.json', JSON.generate([]))
    movies_data
  end
end
```